components/brave_wallet_ui/page/wallet_page_api_proxy.d.ts

```diff
@@ -24,5 +24,5 @@ export default class APIProxy {
   addKeyringControllerObserver: (store: any) => void
   addEthTxControllerObserverObserver: (store: any) => void
   makeTxData: (nonce: string, gasPrice: string, gasLimit: string, to: string, value: string) => any
-  getKeyringsByType: (type: string) => Promise<KeyringController>
+  getKeyringsByType: (type: string) => any
```

Comment:
We should not be exporting an entire API as `any` - this defeats the whole point of type safety.